joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/IdentifierTests.scala

```diff
@@ -165,8 +177,73 @@ class IdentifierTests extends RubyCode2CpgFixture {
         cpg.identifier.name("endvar").l.size shouldBe 2
         cpg.identifier.name("beginbool").l.size shouldBe 1
         cpg.identifier.name("endbool").l.size shouldBe 1
+        cpg.call.name("puts").size shouldBe 1
         cpg.identifier.size shouldBe 6
       }
     }
+
+    "The CPG generated for do block" should {
+      val cpg = code(
+        """
+          |[1, 2, "three"].each do |n|
+          | puts n
+          |end
+          |""".stripMargin,
+        fileName = "doblock.rb"
+      )
+
+      "doblocktest" in {
```

Comment:
A more suggestive name would be ideal. (Here and below)

Reply:
Changed the name